dask_expr/_collection.py

```diff
@@ -864,6 +865,54 @@ def set_index(
             )
         )
 
+    def sort_values(
+        self,
+        by: str | list[str],
+        npartitions: int | None = None,
+        ascending: bool | list[bool] = True,
+        na_position: Literal["first"] | Literal["last"] = "last",
+        partition_size: float = 128e6,
+        sort_function: Callable[[pd.DataFrame], pd.DataFrame] | None = None,
+        sort_function_kwargs: Mapping[str, Any] | None = None,
+    ):
```

Comment:
I don't consider `upsample` to be a critical key-word argument, but was there a specific reason you left it out?

It looks like you would only need to pass it through to `_calculate_divisions`.

Reply:
Yeah I left it out of set_index as well, will do as a follow up. Should probably have done this before doing sort values